content/docs/tech-insights/introduction/index.md

```diff
@@ -13,30 +13,28 @@ Therefore, in order to create [Scorecards](../scorecards/index.md), you will nee
 ## Prerequisites
 
 - Roadie Tech Insights is a paid add-on. If you are not sure about whether or not you have it or would like to include it, please reach out to our sales team.
-- You must be an administrator for your Roadie instance to enable and manage Tech Insights.
+- You must be an administrator for your Roadie instance to manage Tech Insights.
 
-## Enable Roadie Tech Insights
+## READ Access
 
-To enable Roadie Tech Insights, go to Administration → Settings → Plugins → Tech Insights, or navigate to `https://[organisation]/administration/settings/tech-insights` and tick the enabled box as shown below:
+By default, all users will be able to see but not edit Tech Insights content.
 
-![Enable Tech Insights](./enable-tech-insights.png)
+If you want to hide Tech Insights for non-admin users such as when initially setting it up, you can check 'Admins only' checkbox under Administration → Settings → Plugins → Tech Insights page. This will remove Tech Insights from the sidebar for anyone who is not part of `roadie-backstage-admin` nor `roadie-tech-insights-admin` groups.
```

Comment:
```suggestion
If you want to hide Tech Insights for non-admin users such as when initially setting it up, you can check 'Admins only' checkbox under Administration → Settings → Plugins → Tech Insights page. This will remove Tech Insights from the sidebar for anyone who is not part of `roadie-backstage-admin` or `roadie-tech-insights-admin` groups.
```

Reply:
I think nor is correct - "not part of x, nor y".

Reply:
I believe it should be not part of neither x nor y. Without the extra word there the meaning changes to not part of x, "or not y"